src/main/java/org/myrobotlab/service/InMoovTorso.java

```diff
@@ -102,18 +102,10 @@ public void setAutoDisable(Boolean param) {
     lowStom.setAutoDisable(param);
   }
 
-  public void setOverrideAutoDisable(Boolean param) {
-    topStom.setOverrideAutoDisable(param);
-    midStom.setOverrideAutoDisable(param);
-    lowStom.setOverrideAutoDisable(param);
-  }
-
   @Override
   public void broadcastState() {
     // notify the gui
-    topStom.broadcastState();
-    midStom.broadcastState();
-    lowStom.broadcastState();
+    Runtime.broadcastStates();

```

Comment:
same question as before.. why isn't this handled in the base class.

Reply:
the question and answer was never raised - 
does a broadcast of the service mean broadcast of the peers .. it "could" its just never been implemented.